.github/actions/workflow-conclusion/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@technote-space/workflow-conclusion-action",
+  "version": "3.0.3",
+  "description": "GitHub action to get workflow conclusion.",
+  "keywords": [
+    "github",
+    "github actions"
+  ],
+  "homepage": "https://github.com/technote-space/workflow-conclusion-action",
+  "bugs": {
+    "url": "https://github.com/technote-space/workflow-conclusion-action/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/technote-space/workflow-conclusion-action.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Technote",
+    "email": "technote.space@gmail.com",
+    "url": "https://technote.space"
+  },
+  "files": [
+    "lib",
+    "action.yml"
+  ],
+  "scripts": {
+    "build": "rm -rdf lib && rollup -c",
+    "cover": "vitest run --coverage",
+    "postinstall": "[ -n \"$CI\" ] || [ ! -f node_modules/.bin/husky ] || husky install",
+    "lint": "eslint 'src/**/*.ts' --cache",
+    "lint:fix": "eslint --fix 'src/**/*.ts'",
+    "prepublishOnly": "[ -n \"$CI\" ] || [ ! -f node_modules/.bin/pinst ] || pinst --disable",
+    "postpublish": "[ -n \"$CI\" ] || [ ! -f node_modules/.bin/pinst ] || pinst --enable",
+    "release": "yarn release-ga --test",
+    "test": "yarn lint && yarn typecheck && yarn cover",
+    "typecheck": "tsc --noEmit",
+    "update": "npm_config_yes=true npx npm-check-updates -u --timeout 100000 && yarn install && yarn upgrade && yarn audit"
+  },
+  "devDependencies": {
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
+    "@commitlint/cli": "^17.3.0",
+    "@commitlint/config-conventional": "^17.3.0",
+    "@octokit/openapi-types": "^14.0.0",
+    "@octokit/types": "^8.0.0",
+    "@rollup/plugin-commonjs": "^23.0.3",
+    "@rollup/plugin-json": "^5.0.2",
+    "@rollup/plugin-node-resolve": "^15.0.1",
+    "@rollup/plugin-typescript": "^10.0.1",
+    "@sindresorhus/tsconfig": "^3.0.1",
+    "@technote-space/github-action-helper": "^5.3.10",
+    "@technote-space/github-action-log-helper": "^0.2.9",
+    "@technote-space/github-action-test-helper": "^0.11.1",
+    "@technote-space/release-github-actions-cli": "^1.9.3",
+    "@types/node": "^18.11.11",
```

Comment:
Maybe with some follow-up work we ought to use versions that are Node.js v20+ appropriate in case we ever want to repackage.